docs/source/noise/index.rst

```diff
@@ -114,51 +114,51 @@ Noise types for each column
     - Types of noise
     - Notes
   * - First name
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
     - Leave a field blank, use a fake name, nickname, make typos, make OCR errors, make phonetic errors
-    -
+    - In the 1040 form, the same noise types are applied to the corresponding columns for the joint filer and dependents
```

Comment:
```suggestion
    - In the 1040 form, the same noise types apply to the first name columns for the joint filer and dependents
```
Which noise types are actually applied to the columns depends on the configuration specified by the user, so I changed the wording to try to convey that the same noise types are _available_ for these other columns. I'm open to other wording suggestions.

Also, I think it would be clearer to specify which column we mean, since this row in the table applies to a specific column name.